src/core/lib/surface/completion_queue.cc

```diff
@@ -117,8 +117,9 @@ static grpc_error* non_polling_poller_work(grpc_pollset* pollset,
   gpr_timespec deadline_ts =
       grpc_millis_to_timespec(deadline, GPR_CLOCK_MONOTONIC);
   while (!npp->shutdown && !w.kicked &&
-         !gpr_cv_wait(&w.cv, &npp->mu, deadline_ts))
+         !gpr_cv_wait(&w.cv, &npp->mu, deadline_ts)) {
     ;
```

Comment:
No need for this anymore, now that there are braces around it.

Reply:
Done